src/test/kotlin/it/unibo/protelis/web/TestBridgeVerticle.kt

```diff
@@ -0,0 +1,18 @@
+package it.unibo.protelis.web
+
+import io.vertx.core.Vertx
+import io.vertx.junit5.VertxExtension
+import io.vertx.junit5.VertxTestContext
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(VertxExtension::class)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [TestBridgeVerticle] at /src/src/test/kotlin/it/unibo/protelis/web/TestBridgeVerticle.kt:10:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=5040950)